### PR TITLE
Adding Kamlesh nagware as Lab Steward

### PR DESCRIPTION
**Pull request by knagware9**
Adding Kamlesh nagware as Lab Steward

Signed-off-by: Kamlesh Nagware <kamlesh.nagware@yahoo.com>

## Checklist for Hyperledger Lab Proposal PR

- [ ] Read [README.md](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/README.md) carefully
- [ ] Make sure your changes are committed with the proper sign-off
- [ ] Use the labs name as the PR title
- [ ] Copy the short description of the proposal as the description of the PR
- [ ] Remove this checklist section
- [ ] Submit your PR and the labs stewards will review your proposal
- [ ] Ask your sponsor to confirm sponsorship by stating so in a comment to the PR

### CODE REVIEW

**Comment by vipinsun** [comment]
Was this brought up in the TSC or in conversations? @tkuhrt @lehors - I do not have any objections to this as we are down to three lab stewards.

File: stewards.md [hunk 1]
```diff
@@ -8,10 +8,12 @@ Stewards
 | Arnaud J Le Hors | [lehors][lehors] |
 | Tracy Kuhrt | [tkuhrt][tkuhrt] |
 | Vipin Bharathan | [vipinsun][vipinsun] |
+| Kamlesh Nagware | [knagware9][knagware9]
```
Comment (tkuhrt) [inline stewards.md:11]:
```suggestion
| Kamlesh Nagware | [knagware9][knagware9] |
```

Comment (ryjones) [reply stewards.md:11]: I've been sorting these by GitHub username - please take Tracy's suggested edit, and move this to the top of the list

**Review by ryjones — CHANGES_REQUESTED** [review]
Please sort by GitHub alias and apply Tracy's suggestion

**Review by ryjones — APPROVED** [review]

**Comment by ryjones** [comment]
Merged

**Pull request closed without merging**